rta/hosts_file_modify.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+# Name: Hosts File Modified
+# RTA: hosts_file_modify.py
+# ATT&CK: T1492
+# Description: Modifies the hosts file
+
+import os
+import random
+import time
+
+from string import ascii_letters
+
+from . import common
+
+
+def main():
+    hosts_files = {
+        common.WINDOWS: "C:\\Windows\\system32\\drivers\\etc\\hosts",
+        common.LINUX: "/etc/hosts",
+        common.MACOS: "/private/etc/hosts"
+    }
+    hosts_file = hosts_files[common.CURRENT_OS]
+
+    backup = os.path.abspath(hosts_file + '_backup')
+    common.log("Backing up original 'hosts' file.")
+    common.copy_file(hosts_file, backup)
+
+    # add randomness for diffs for FIM module
+    randomness = ''.join(random.sample(ascii_letters, 10))
+    entry = [
+        '',
+        '# RTA hosts_modify was here',
+        # '8.8.8.8 https://www.{random}.google.com'.format(random=randomness)
```

Comment:
```suggestion
        '# 8.8.8.8 https://www.{random}.google.com'.format(random=randomness)
```

My mistake from earlier

At the moment, the rule is solely looking for modification of the file itself. If down the road we need to emulate an actual processing of a domain lookup defined within, we can revisit